### ISSUE TITLE
Enhance Save State Management: Delete

### ISSUE THREAD

**Issue by bengalih** [hrydgard/ppsspp#16223]
### What should happen

Ability to selectively delete a save state in this screen.
While you can always just overwrite an existing save, the ability to clear/delete a save is also beneficial.

This could be less important if my request https://github.com/hrydgard/ppsspp/issues/16222 was implemented as we could also then just rename a save as "UNNEEDED" or something similar.

### Who would this benefit

Anyone who saves state often and in multiple places.

### Platform (if relevant)

Android

### Games this would be useful in

ALL

### Checklist

- [X] Check the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already implemented.
- [X] [Search for other requests](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same feature.

**Comment by Gamemulatorer** [comment]
This is still not implemented?

**Comment by hrydgard** [comment]
I don't think so. Tagging for looking at in the next release.

**Comment by unknownbrackets** [comment]
You can delete save states under the "Savedata manager" in Settings -> Tools.  It's true we could cram a ton of options on the Pause screen, but that only sounds like a good idea until you realize you're looking at something like this:
![image](https://user-images.githubusercontent.com/191233/206840427-04abe35b-2951-4097-9399-688790014a84.png)

Currently tapping the save state image already shows 4 buttons: overwrite, load, close, and undo.  I guess we could replace back with delete.  Or maybe use a swiping gesture on the list.  Maybe a better way would be to let you go to the savedata manager, though.

-[Unknown]

**Comment by hrydgard** [comment]
Hehe. Yeah by looking at it I didn't necessarily mean to put delete buttons directly on the savedata screen, just to improve the accessibility of it somehow, and that could indeed be an easier way to get to the manager.

**Comment by VL4DST3R** [comment]
Is this still not part of the emulator almost 3 years later? How come we have superfluous (even if nice) QOL features like "undo last save" before actually being able to delete them?

**Comment by hrydgard** [comment]
Well, mostly because this is the only request that has ever been made for this feature. More popular requests and serious game issues are prioritized.

**Comment by VL4DST3R** [comment]
Fair enough, I just figured that even during testing the need to delete or otherwise manage them came up organically. Here's hoping it will be added soon.

**Comment by hrydgard** [comment]
#20718 implements this.
